context/app/static/js/components/detailPage/Protocol/Protocol.jsx

```diff
@@ -36,11 +37,17 @@ function ProtocolLink({ url, index }) {
 }
 
 function Protocol({ protocol_url }) {
+  const {
+    entity: { entity_type },
+  } = useFlaskDataContext();
+
   const protocolUrls = useFormattedProtocolUrls(protocol_url, 1);
 
   return (
     <DetailPageSection id="protocols">
-      <SectionHeader>Protocols</SectionHeader>
+      <SectionHeader iconTooltipText={`Protocols provided by protocols.io for the given ${entity_type}`}>
```

Comment:
Can you change the text to `Protocols uploaded to protocols.io for the given {entity_type}`? Thanks!

Reply:
Would we want lowercase here too?